content/javascript/concepts/strings/terms/slice/slice.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: '.slice()'
+Description: 'Returns a selected portion of a string.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Elements'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `.slice()` method selects and returns a selected portion of a string.
+
+## Syntax
+
+```pseudo
+string.slice(start);
+
+string.slice(start, stop);
+```
+
+- The `start` index is optional and begins at zero (inclusive) by default.
+- The `stop` index is also optional and defaults to the length of the string (not inclusive).
+
+## Example
+
+In the example below, a `smoothie` string is created. Next, the `.slice()` method is used to extract substrings for the berry variable:
+
+```js
+let smoothie = "mango strawberry pineapple";
+
+let berry = smoothie.slice(6, 16);
+
+console.log(smoothie);
+// Output: mango strawberry pineapple
```

Comment:
These comments are redundant with the output listed below. I'd remove them.

Reply:
I agree. Thank you